src/ComboControls.ts

```diff
@@ -249,11 +249,8 @@ export default class ComboControls extends EventDispatcher {
     }
 
     const { domElement } = this;
-    let { x, y } = getHTMLOffset(
-      domElement,
-      event.clientX,
-      event.clientY,
-    );
+    let x = event.offsetX;
```

Comment:
I am not sure that `offset(X/Y)` is always defined for all browsers

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/offsetX#Browser_compatibility

All but mobile (which uses touch anyway?)